### PR TITLE
Add user namespace to create repo modal

### PR DESCRIPTION
**Pull request by syed**
image

### CODE REVIEW

**Comment by syed** [comment]
Also added input validation
image

**Review by HammerMeetNail — APPROVED** [review]
LGTM

File: src/components/modals/CreateRepoModalTemplate.tsx [hunk 1]
```diff
@@ -213,4 +228,5 @@ type CreateRepositoryModalTemplateProps = {
   isModalOpen: boolean;
   handleModalToggle?: () => void;
   orgName?: string;
+  updateListHandler: (value: IRepository) => void;
```
Comment (harishsurf) [inline src/components/modals/CreateRepoModalTemplate.tsx:231]: don't think we use this anywhere

Comment (syed) [reply src/components/modals/CreateRepoModalTemplate.tsx:231]: what about this one? https://github.com/quay/quay-ui/blob/main/src/routes/RepositoriesList/RepositoriesList.tsx#L260

Comment (harishsurf) [reply src/components/modals/CreateRepoModalTemplate.tsx:231]: yeah, it's a dead code. I forgot to remove it in my earlier PR.

**Pull request merged**